docs/modules/ROOT/pages/how-to-guides/proc_release_advisory.adoc

```diff
@@ -0,0 +1,55 @@
+= Releasing with an advisory
+
+In order to be able to create an advisory when releasing a Snapshot, certain data has to be included in the Release Plan Admission.
+
+.*Prerequisites*
+
+* You have an existing Release Plan Admission for your release.
+
+.*Procedures*
+
+. Add the advisory data in your ReleasePlanAdmission object.
+
+*Example `ReleasePlanAdmission.yaml` object*
+
+[source,yaml]
+----
+apiVersion: appstudio.redhat.com/v1alpha1
+kind: ReleasePlanAdmission
+metadata:
+  ..
+spec:
+ data:
+   releaseNotes: <advisory data> <.>
```

Comment:
There's not much information here for a user to go on. We should flesh this out more in a followup PR.